cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporter.java

```diff
@@ -332,10 +335,30 @@ public void onCompletion(RecordMetadata recordMetadata, Exception e) {
     });
   }
 
+  /**
+   * Starting with Kafka 2.6.0, a new class KafkaYammerMetrics provides the default Metrics Registry. The old default
+   * registry does not work with 2.6+. Therfore if the new class exists, we use it and if it doesn't exist we use the
+   * old one. More details can be found here: https://github.com/apache/kafka/blob/2.6.0/core/src/main/java/kafka/metrics/KafkaYammerMetrics.java
+   *
+   * Once CC supports only 2.6.0 and newer, we can clean this up and use only KafkaYammerMetrics all the time.
```

Comment:
I initially assumed there were significant backwards incompatible changes from 2.5 to 2.6, but if this function would be sufficient to make `migrate_to_kafka_2_5` compatible with `Kafka 2.6`, can we rebase this patch on `migrate_to_kafka_2_5` branch to eliminate the overhead of managing one more branch?

Reply:
I will run some tests to make sure this metrics reporter works ok with 2.5.

Reply:
From my testing it seems this function works as expected and allows this code to be run on Kafka 2.5 and 2.6 brokers. So I will rebase this against the `migrate_to_kafka_2_5` branch and you can delete the 2.6 branch.